src/routes/profile.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { Avatar } from "../components/avatar/avatar";
+import { Pill } from "../components/pill/pill";
+
+/**
+ * Profile Route
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param src the avatar image link
+ * @param alt the avatar image description
+ * @param name the account holder's name
+ * @param pillText the text to be displayed in the pill
+ * @param isCopiable (refers to pill) 'true' or 'false'
+ * @param email account associated email
+ * @param id account ID
+ */
+
+const Container = styled.div`
+  margin: 40px auto;
+  max-width: 250px;
+  row-gap: 12px;
+`;
+
+export const Profile = ({
+  src,
```

Comment:
I think all of these props will actually come from the SDK once we implement that; for now, could you put a state variable inside this component for `profileInfo` and `isLoading` and then put a `useEffect` hook that just sets those variables to the values that you are currently passing into this component from `main.jsx`? Then put a `// TODO` to indicate that is where the API call will go for fetching the profile info?

```
const [profileInfo, setProfileInfo] = useState();
const [isLoading, setIsLoading] = useState(true);

useEffect(() => {
   // TODO: Replace this with the API call for fetching profile data
   setProfileInfo({/*...*/})
   setLoading(false)
}, []);
```

Then inside of the component, you can put in some nice loading indicator and render it as long as `isLoading` is true

https://react-bootstrap.github.io/components/spinners/